.cspell/cspell.json

```
@@ -0,0 +1,57 @@
+{
+  "$schema": "https://raw.githubusercontent.com/streetsidesoftware/cspell/main/cspell.schema.json",
+  "version": "0.2",
+  "dictionaryDefinitions": [
+    {
+      "name": "backdrop",
+      "path": "backdrop.dic"
+    },
+    {
+      "name": "drupal",
+      "path": "https://git.drupalcode.org/project/drupal/-/raw/10.1.x/core/misc/cspell/dictionary.txt"
```

Can we please change this to be using the `HEAD` keyword, which gets the default branch every time?:
```suggestion
      "path": "https://git.drupalcode.org/project/drupal/-/raw/HEAD/core/misc/cspell/dictionary.txt"
```